.github/workflows/flambda-backend.yml

```diff
@@ -89,7 +89,7 @@ jobs:
       working-directory: merlin-jst
       run: |
         opam depext conf-jq --yes # opam depext bug
-        opam pin menhirLib 20210419 --no-action
+        opam pin menhirLib 20231231 --no-action
```

Comment:
This change is the opposite of the one described in the PR description. My hunch is that the code is correct while the description is wrong, but please confirm.

Reply:
That's correct. I pushed this PR to initially do the thing in the description, but then decided to instead bump the version. So I pushed the updated code, then updated the description. You must've looked at the PR between me pushing and updating the description. Sorry about that.